### PR TITLE
Update cluster migration scenario with more details

### LINKED ISSUE #2167: Cluster Migration Documentation not clear - Unusable
**Describe the problem/challenge you have**

The documentation link on cluster migration here:

https://velero.io/docs/master/migration-case/

a) Does not indicate how to specify the new backup location when restoring a backup from the newly created backup location that points to the-cluster-that-must-be-migrated. Should it be passed as an argument to the "velero create restore from <backup> ...." command?

Because of this the attempt to migrate the cluster stops at the point where the second backup location pointing to the source cluster's container is created. There does not seem to be a way to indicate that the backup desired must come from the newly created backup location.

b) There does not seem to be clear documentation on specifying additional backup locations for each cloud vendor (Azure in my case). This makes the migration process even more uncertain.

Note: This test is being carried out on azure cloud.

**Describe the solution you'd like**

- Demonstrate using exact commands in the documentation how the second backup location is created, including the authentication configuration, for each of the major cloud vendors (Azure, AWS, etc ...)
- Demonstrate the exact command(s) use to restore a backup from a particular storage location.

**Anything else you would like to add:**

The velero command to restore from a particular backup doesn't seem to include an option to specify the exact backup location (container) from which the backup must be extracted. It's hard to understand how the documented migration process will work without this feature.

Some clarification would be appreciated here.

**Environment:**

- Velero version (use `velero version`):

v1.1.0

- Kubernetes version (use `kubectl version`):

1.14.8

- Kubernetes installer & version:

AKS 1.14.8

- Cloud provider or hardware configuration:

Azure, Azure blob storage

- OS (e.g. from `/etc/os-release`):

Ubuntu Linux

### PR DESCRIPTION
**Pull request by a-mccarthy**
Signed-off-by: Abigail McCarthy <mabigail@vmware.com>

Thank you for contributing to Velero!

# Please add a summary of your change

This updates the cluster migration scenario docs to include more details about how to set up clusters to data migration.

# Does your change fix a particular issue?

Fixes #2167 

# Please indicate you've done the following:

- [x] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [x] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [x] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

File: site/content/docs/main/migration-case.md [hunk 1]
```diff
@@ -3,52 +3,85 @@ title: "Cluster migration"
...
+
+    The default backup retention period, expressed as TTL (time to live), is 30 days (720 hours); you can use the `--ttl <DURATION>` flag to change this as necessary. See [how velero works](how-velero-works.md#set-a-backup-to-expire) for more information about backup expiry.
+
+1. On Cluster 2, make sure that Velero is installed. Note that the install command below has the same `region` and `--bucket` location as the install command for Cluster 1. The Velero plugin for AWS does not support migrating data between regions.
+
+    ```
+    velero install --provider aws --image velero/velero:v1.8.0 --plugins velero/velero-plugin-for-aws:v1.4.0 --bucket velero-migration-demo --secret-file xxxx/aws-credentials-cluster2 --backup-location-config region=us-east-2 --snapshot-location-config region=us-east-2
+    ```
+
+    Alternatively you could configure `BackupStorageLocations` or `VolumeSnapshotLocations` after installing Velero on Cluster 2, pointing to the `--bucket` location used by Cluster 1. To do this you can use to `velero backup-location create` and `velero snapshot-location create` commands.
```
Comment (a-mccarthy) [inline site/content/docs/main/migration-case.md:55]: @reasonerjt Should I remove the VolulmneSnapShot information? This is the only place that its mentioned in this doc, but i'm not sure if the process would be the same. thanks!

**Review by blackpiglet — DISMISSED** [review]

File: site/content/docs/main/migration-case.md (hunk 1 above)
Comment (reasonerjt) [reply site/content/docs/main/migration-case.md:55]: The `VolumeSnapshotLocation` should be created, thanks for asking.

**Comment by reasonerjt** [comment]
I noticed there are several instances where you refer to `backup` as `back up`, is it intentional?

File: site/content/docs/main/migration-case.md [hunk 2]
```diff
@@ -3,52 +3,85 @@ title: "Cluster migration"
...
 
     ```
     velero backup describe <BACKUP-NAME>
     ```
 
-    **Note:** The default sync interval is 1 minute, so make sure to wait before checking. You can configure this interval with the `--backup-sync-period` flag to the Velero server.
+    Velero resources are [synchronized](how-velero-works.md#object-storage-sync) with the backup files in object storage. This means that the Velero resources created by Cluster 1's backup will be synced to Cluster 2 through the shared Backup Storage Location. Once the sync occurs, you will be able to access the back up from Cluster 1 on Cluster 2 using Velero commands. The default sync interval is 1 minute, so you may need to wait before checking for the back up's availability on Cluster 2. You can configure this interval with the `--backup-sync-period` flag to the Velero server on Cluster 2.
 
-1.  *(Cluster 2)* Once you have confirmed that the right Backup (`<BACKUP-NAME>`) is now present, you can restore everything with:
+1.  On Cluster 2, once you have confirmed that the right back up is available, you can restore everything to Cluster 2.
```
Comment (reasonerjt) [inline site/content/docs/main/migration-case.md:72]: Is it `backup` or `back up`?

There are several other instances like this in this PR.

File: site/content/docs/main/migration-case.md (hunk 1 above)
Comment (reasonerjt) [reply site/content/docs/main/migration-case.md]: that is correct.

**Review by reasonerjt — APPROVED** [review]
LGTM

**Review by blackpiglet — APPROVED** [review]

**Pull request merged**